_posts/2018-08-13-Introducing....md

```diff
@@ -0,0 +1,96 @@
+---
+layout: post
+title: ### Introducing...
+categories: blog
+excerpt: ### blog introducing three people from Kirstie's lab 
+tags: [] ### INSERT TAGS IF APPLICABLE
+image:
+  feature:
+link: 
+date: 2019-08-13 ### UPDATE date
+modified:
+share: true
+author: GeorgiaHCA ###
+---
+
+:tada: :star2: :rocket: Introducing three of my stellar Turing colleagues... 
+
+## :star: Sarah Gibson :star: 
+
+#### "Tell me about your background"
+
+For her PhD in Astrophysics, Sarah worked on a NASA (:heart_eyes:!!) project called Swift hunting down gamma ray bursts. Gamma ray bursts are very brief flashes of high energy radiation, and it’s difficult to predict when and where they will go off. 
+
+Gamma ray bursts form in two ways: when young massive stars go supernova, or two neutron stars merge together. The first produces gamma ray bursts which are long in time duration because they have a lot of matter. They happen in very early galaxies and are very bright, and studying them tells us a lot about the first galaxies that were formed after the Big Bang. Many galaxies may not be bright enough to see without these gamma rays, so studying them can help us discover new galaxies, as well as telling us the composition of stuff between us and the burst - e.g. what our galaxy is made of, what the galaxy the burst happened in is made of, and what the stuff in between is made of.
```

Comment:
```suggestion
Gamma ray bursts form in two ways: when young massive stars go supernova, or two neutron stars merge together. The first produces gamma ray bursts which are long in time duration because they have a lot of matter. They happen in very early galaxies and are very bright, and studying them tells us a lot about the first galaxies that were formed after the Big Bang. Many galaxies may not be bright enough to see without these gamma rays, so studying them can help us discover new galaxies, as well as telling us the composition of stuff between us and the burst - e.g. what our Galaxy is made of, what the galaxy the burst happened in is made of, and what the stuff in between is made of.
```
Very tiny, nit-picky point. The convention is to capitalise "galaxy" when referring to the one we live in 😊